README.md

```diff
@@ -106,3 +106,12 @@ Getting Started on AWS
     --root-drive=hello-rootfs.ext4 \
     --kernel-opts="console=ttyS0 noapic reboot=k panic=1 pci=off nomodules rw"
   ```
+
+Testing
+---
+
+Most tests do not need access to Firecracker or the KVM. However, the
```

Comment:
How do I skip the tests that require access to a Firecracker binary or KVM?

Reply:
Tests are skip by default if the binary hasn't been compiled and if no kernel image is present. I can update the doc to reflect that or we can change the behavior where you opt in to skipping rather than skipping by default.

Reply:
I'd prefer to have opt-in to skipping.  In general I don't typically like auto-skipped tests as someone may not notice that the test is skipped and then have a false sense of test coverage.

Reply:
It looks like you've added a `SKIP_INTEG_TEST` environment setting. Please document it here.